Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -60,111 +100,158 @@ public extension ActionButton {
   }
 }
 
-// MARK: - Private Methods
-private extension ActionButton {
-  func buttonAction() {
-    actionHandler?()
-  }
-  
-  func getAlignment(for extraImage: ExtraImage?) -> Alignment {
-    switch extraImage {
-    case .some(let alignment):
-      switch alignment {
-      case .right:
-        return .trailing
-      case .left:
-        return .leading
-      case .center:
-        return .center
-      }
-    case .none:
-      return .trailing
-    }
+// MARK: - Builder Pattern Methods
+@available(iOS 13, *)
+public extension ActionButton {
+  func title(_ title: String) -> ActionButton {
+    self.properties.title = title
+    return self
+  }
+  
+  func font(_ font: Font) -> ActionButton {
+    self.properties.font = font
+    return self
+  }
+  
+  func textColor(_ textColor: Color) -> ActionButton {
+    self.properties.textColor = textColor
+    return self
+  }
+  
+  func cornerRadius(_ cornerRadius: ActionButton.CornerRadiusType) -> ActionButton {
+    self.properties.cornerRadius = cornerRadius
+    return self
+  }
+  
+  func backgroundType(_ backgroundType: ActionButton.Background) -> ActionButton {
+    self.properties.backgroundType = backgroundType
+    return self
+  }
+  
+  func borderColor(_ borderColor: Color) -> ActionButton {
+    self.properties.borderColor = borderColor
+    return self
+  }
+  
+  func borderWidth(_ borderWidth: CGFloat) -> ActionButton {
+    self.properties.borderWidth = borderWidth
+    return self
+  }
+  
+  func extraImage(leftImage: ActionButton.ExtraImage) -> ActionButton {
+    self.properties.leftImage = leftImage
+    return self
+  }
```

Comment:
Why don't we simply name this method as `leftImage`?

Reply:
You're right, sounds better. I'll rename all 4 methods responsible for the images accordingly.